Client/package.json

```diff
@@ -16,7 +16,8 @@
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
-    "selenium-webdriver": "^4.0.0-alpha.7"
+    "selenium-webdriver": "^4.0.0-alpha.7",
+    "styled-components": "^5.2.1"
```

Comment:
This was removed in develop

Reply:
Yes, this is an unintentional merge conflict that didn't come up. Will fix